src/locales/en-US/stream.ftl

```diff
@@ -85,6 +85,9 @@ comments-permalinkView-singleConversation = SINGLE CONVERSATION
 comments-inReplyTo = In reply to <username></username>
 comments-replyTo = Replying to: <username></username>
 
+comments-reportButton-report = Report
+comments-reportButton-reported = Reported
+
 ## Profile Tab
 profile-historyComment-viewConversation = View Conversation
 profile-historyComment-replies = Replies {$replyCount}
```

Comment:
Obviously unrelated to this PR, but could you fix this so when there is one reply, it doesn't read as `Replies 1`?